examples/events.rs

```diff
@@ -0,0 +1,27 @@
+#[macro_use]
+extern crate redis_module;
+
+use redis_module::{Context, raw};
+
+fn on_event(ctx: &Context, event_type: raw::NotifyEvent, event: &str, key: &str) {
+    let msg = format!("Received event: {:?} on key: {} via event: {}", event_type, key, event);
+    ctx.log_debug(msg.as_str());
+}
+
+//////////////////////////////////////////////////////
+
+redis_module! {
+    name: "events",
+    version: 1,
+    data_types: [],
+    commands: [],
+    event_handlers: [
+        [raw::NotifyEvent::EXPIRED | raw::NotifyEvent::EVICTED, on_event]
```

Comment:
The only thing I dislike about this is the fact that the `raw` module is leaking to the highest-level macro definitions. As an alternative, we could have a "higher-level" enum mapped to this defined in `redismodule.rs`, but for the PoC I decided to go with the minimal approach.